### PR TITLE
[FLINK-16101][docs-zh] Translate "Hive Functions" page of "Hive Integration" into Chinese

### PR DESCRIPTION
**Pull request by zzh1985**
## What is the purpose of the change

Translate the Hive functions doc to Chinese
## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: ( no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): ( no )
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: ( no)

## Documentation

  - Does this pull request introduce a new feature? ( no)
  - If yes, how is the feature documented? (docs )

### CODE REVIEW

**Comment by wuchong** [comment]
Wrong issue number? I think it should be FLINK-16101?

**Comment by wuchong** [comment]
Btw, please request assignee before open a pull request.

**Comment by zzh1985** [comment]
Sorry for that，the title of the issue contains two issue number like this :
Flink / FLINK-16080 Translate "Table API & SQL" pages into Chinese / FLINK-16101
I got the wrong number.
https://issues.apache.org/jira/browse/FLINK-16101?src=confmacro

**Comment by zzh1985** [comment]
> Btw, please request assignee before open a pull request.

How to request assignee, Sorry I'm a newbie.

**Comment by zzh1985** [comment]
> > Btw, please request assignee before open a pull request.
> 
> How to request assignee, Sorry I'm a newbie.

In the https://issues.apache.org/jira/browse/FLINK-16101 and https://github.com/apache/flink page,  I can't find the assignee entrance.

**Comment by wuchong** [comment]
Hi @zzh1985, you can leave a comment under FLINK-16101 issue page.

File: docs/dev/table/hive/hive_functions.zh.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-## Use Hive Built-in Functions via HiveModule
+## 通过 HiveModule 使用 Hive 内置函数
 
-The `HiveModule` provides Hive built-in functions as Flink system (built-in) functions to Flink SQL and Table API users.
+在Flink SQL 和 Table API 中，可以通过系统内置的`HiveModule` 来使用 Hive 内置函数，
```
Comment (klion26) [inline docs/dev/table/hive/hive_functions.zh.md:27]:
```suggestion
在 Flink SQL 和 Table API 中，可以通过系统内置的`HiveModule` 来使用 Hive 内置函数，
```

File: docs/dev/table/hive/hive_functions.zh.md [hunk 2]
```diff
@@ -22,11 +22,11 @@ specific language governing permissions and limitations
...
 
-For detailed information, please refer to [HiveModule]({{ site.baseurl }}/dev/table/modules.html#hivemodule).
+详细信息，请参考 [HiveModule]({{ site.baseurl }}/dev/table/modules.html#hivemodule)。
```
Comment (klion26) [inline docs/dev/table/hive/hive_functions.zh.md:29]:
```suggestion
详细信息，请参考 [HiveModule]({{ site.baseurl }}/zh/dev/table/modules.html#hivemodule)。
```

File: docs/dev/table/hive/hive_functions.zh.md [hunk 3]
```diff
@@ -58,34 +58,33 @@ modules:
 </div>
 </div>
 
-* NOTE that some Hive built-in functions in older versions have [thread safety issues](https://issues.apache.org/jira/browse/HIVE-16183).
-We recommend users patch their own Hive to fix them.
+* 请注意在旧版本的部分 Hive 内置函数存在 [线程安全问题](https://issues.apache.org/jira/browse/HIVE-16183)。
```
Comment (klion26) [inline docs/dev/table/hive/hive_functions.zh.md:61]:
```suggestion
* 请注意旧版本的部分 Hive 内置函数存在 [线程安全问题](https://issues.apache.org/jira/browse/HIVE-16183)。
```

File: docs/dev/table/hive/hive_functions.zh.md [hunk 4]
```diff
@@ -184,8 +183,7 @@ myudtf
 
 {% endhighlight %}
 
-
-Then, users can use them in SQL as:
+此时，用户如果想使用这些UDF，在 SQL 中就可以这样写：
```
Comment (klion26) [inline docs/dev/table/hive/hive_functions.zh.md:186]:
```suggestion
此时，用户如果想使用这些 UDF，在 SQL 中就可以这样写：
```

File: docs/dev/stream/state/state.zh.md [hunk 5]
```diff
@@ -472,7 +472,7 @@ val counts: DataStream[(String, Int)] = stream
 
 ## 使用 Managed Operator State
 
-用户可以通过实现 `CheckpointedFunction` 或 `ListCheckpointed<T extends Serializable>` 接口来使用 managed operator state。
```
Comment (klion26) [inline docs/dev/stream/state/state.zh.md:475]: 把这个文档的修改 revert 回去吧，这个 PR 仅修改 hive function 的翻译，改完后可以 ping 我

File: docs/dev/table/hive/hive_functions.zh.md [hunk 6]
```diff
@@ -58,39 +58,38 @@ modules:
 </div>
 </div>
 
-* NOTE that some Hive built-in functions in older versions have [thread safety issues](https://issues.apache.org/jira/browse/HIVE-16183).
-We recommend users patch their own Hive to fix them.
+* 请注意旧版本的部分 Hive 内置函数存在 [线程安全问题](https://issues.apache.org/jira/browse/HIVE-16183)。
```
Comment (klion26) [inline docs/dev/table/hive/hive_functions.zh.md:61]:
```suggestion
* 请注意旧版本的部分 Hive 内置函数存在[线程安全问题](https://issues.apache.org/jira/browse/HIVE-16183)。
```

**Comment by zzh1985** [comment]
@klion26 按您的意思修改了，你再帮忙看一下，另外弱弱问一下“ping”是怎么一个操作，我这样算吗？

**Pull request closed without merging**